docs/cloud/index.md

```diff
@@ -2,8 +2,14 @@
 id: index
 title: Temporal Cloud documentation
 sidebar_label: Temporal Cloud
+description: Temporal Cloud is a managed, hosted Temporal environment that provides a platform for Temporal Applications—an alternative to deploying and operating your own Temporal Cluster.
+toc_max_heading_level: 4
 ---
 
+<!-- THIS FILE IS GENERATED. DO NOT EDIT THIS FILE DIRECTLY -->
+
+Temporal Cloud is a managed, hosted Temporal environment that provides a platform for Temporal Applications—an alternative to deploying and operating your own Temporal Cluster.
```

Comment:
I'm not sure where this paragraph came from. (It duplicates the following paragraph, except for the links.)